### DISCUSSION TITLE
von willibrand factor

### DISCUSSION THREAD

**Discussion by mojtabaAmiraslanpour** [UvaCsl/HemoCell#105] [General discussion]
Hello all,

I just saw this inside the code and was curious about it.

"(HemoCell) (AddCellType) Error STRING_FROM_VERTEXES not supported anymore."

Can I ask why it is not supported anymore? Any hints on how to implement it?

Best,
Mojtaba

**Comment by gzavo** [comment]
Hi @mojtabaAmiraslanpour,

That was an early try to implement VWF chains inside HemoCell.
It was abandoned in favor of another solution (that is sadly not open-source yet, since we have not yet published with it).
In the git history you can probably find a version when this output was operational, but it is very inaccurate. This is because the IBM kernel size is 0.5 um, resulting in a much-much larger effect volume than what the actual volume the VWF should influence.
